autogen/main/cluster.tf.tmpl

```diff
@@ -273,6 +273,10 @@ resource "google_container_cluster" "primary" {
         enabled = gke_backup_agent_config.value.enabled
       }
     }
+
+    config_connector_config {
```

Comment:
Just a thought @ericyz, it might be possible to use a dynamic block to avoid breaking change for non-beta clusters.  Something like this: 

```
    dynamic "config_connector_config" {
      for_each = var.config_connector ? [{ enabled = true }] : []

      content {
        enabled = true
      }
    }
 ```
 
 LMKWYT - Would require additional changes for the beta clusters and tests, so I'm comfortable with this path as well.

Reply:
Good suggestion. It's a good approach without waiting for next release